contents/blog/how-we-built-email-onboarding.md

```diff
@@ -0,0 +1,132 @@
+>
+> - **Open rate:** Anything above 40% is OK. 50% is the goal.
+> - **Click-through-rate:** Anything above 4% is OK. 6% is the goal. 
+> - **Conversion rate:** Anything above 3% is OK. 5% is the goal. 
+> - **Unsubscribe rate:** Anything below 1% is OK. 0% is the goal.
+
+## Onboarding 3.1: The one where we added personalization
+
+Data is important, but I strongly believe it doesn’t tell the whole story. You also need feedback and intuition. Most of the feedback on our emails came in [3.1](https://github.com/PostHog/meta/issues/127), when we started sending personalized recommendations to users from [my personal email](mailto:joe@posthog.com).
+
+How this works is simple: when a user signs up, they can optionally tell us what their role is. If their `role_at_organization` = `engineer` we trigger an email from me telling them about [our Product for Engineers newsletter](https://newsletter.posthog.com/). 
+
+![Onboarding 3.1](../images/blog/onboarding/onboarding_31.png)
+<caption>Onboarding 3.1 was when we really started tailoring emails based on what we knew about users</caption>
+
+This email performed very well, with a 68% open rate and a 16% CTR. Two emails into the flow and developers were already opting in to _more_ emails! 
+
+Best of all though, because it was personalized and came from me directly, it also earned a steady trickle of replies. I responded in kind and was able to feed in further improvements, including adding another option for `role_at_organization` = `founder`. 
+
+## Onboarding 3.3: The one where we added experiments
+
+In [3.3](https://github.com/PostHog/meta/issues/140) we started running a series of tests and experiments. We  removed the `founder` targeted email, which invited teams to join [PostHog for Startups](/startups), because it was _too_ successful and cannibalised revenue.
+
+Most of the tests we ran proved inconclusive, or outright failed. Subtle changes to subject lines or body copy had no noticeable impact. We learned that users did _slightly_ prefer [Lottie’s](https://posthog.com/community/profiles/227) hedgehog art over screenshots of the app – but the difference was a negligible 1-2% in CTR. 
+
+One successful experiment we did roll out was adding more personalized emails for non-technical users. If `role_at_organization` = `marketing`, `sales` or `product` _and_ a user had not created an [action](/docs/data/actions) within 12 hours, I’d send them another personal tip. These emails averaged a steady 6% CTR and 54% open rate, so we kept them in, but interestingly I didn't see many replies on these messages.
+
+## Onboarding 4.0: The one where we filtered by ICP
+
+[4.0](https://github.com/PostHog/meta/pull/150/) was a massive step up in complexity as a result of our growing number of features. By late 2023 the flow had 28 separate emails, of which users would get a maximum of 6 over several weeks. We still worried this would be too many, but the data suggests otherwise.
```

Comment:
```suggestion
[4.0](https://github.com/PostHog/meta/pull/150/) was a massive step up in complexity as a result of our growing number of features. By late 2023 the flow had 28 separate emails, of which users would get a maximum of 6 over several weeks. We still worried this would be too many, but the low (0.4%) unsubscribe rate suggested otherwise.
```